gradle.properties

```diff
@@ -6,10 +6,10 @@ android.useAndroidX=true
 # Config
 minSdk=14
 targetSdk=31
-coilVersion=1.4.0
+coilVersion=2.0.0-rc01
 
 GROUP=com.commit451.coiltransformations
-VERSION_NAME=1.1.1
+VERSION_NAME=1.0.0
```

Comment:
Should this be `2.0.0`?

Reply:
The way I have CI set up, it doesn't actually read this value, it is overwritten by CI.